src/components/large-fact-card.js

```diff
@@ -80,20 +93,37 @@ export default function LargeFactCard({factTitle, factImage, factSource, factHTM
         />
       </CardContent>
 
+      <Grid justify="space-between" container alignItems="flex-end" className={classes.separatedlineTags}>
+        <Grid item>
+          {factTags.map((factTag, index) => (
+            <Tag key={index} tagLabel={factTag.name} />
+          ))}
+        </Grid>
+      </Grid>
+
       <CardActions disableSpacing>
-        <Grid justify="space-between" container alignItems="flex-end">
+        <Grid justify="space-between" container alignItems="flex-end" wrap="nowrap">
+
+          <Grid item className={classes.inlineTags}>
+            {factTags.map((factTag, index) => (
+              <Tag key={index} tagLabel={factTag.name} />
+            ))}
+          </Grid>
+
           <Grid item>
-            {!isSSR && navigator.share &&
-              <Button color="primary" aria-label="share" onClick={shareClick}>
-                Share
-              </Button>
-            }
+          {!isSSR && navigator.share &&
+                <IconButton aria-label="share" onClick={shareClick}>
+                  <ShareIcon/>
+                </IconButton>
+              }
           </Grid>
-            <Grid item>
-              <IconButton aria-label="fact info" onClick={handleDialogClickOpen}>
-                <InfoOutlinedIcon />
-              </IconButton>
-            </Grid>
+
+          <Grid item>
+            <IconButton aria-label="fact info" onClick={handleDialogClickOpen}>
+              <InfoOutlinedIcon />
+            </IconButton>
+          </Grid>
```

Comment:
This should all be in a single `<Grid item>` or we get this:

![image](https://user-images.githubusercontent.com/1300395/86565397-5d6bc980-bf2d-11ea-8827-d7e0d18c3a66.png)